ports/azure-storage-cpp/vcpkg.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "azure-storage-cpp",
+  "version-string": "7.5.0",
```

```suggestion
  "version-semver": "7.5.0",
```
This should be served by [version-semver](https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md#version-semver).

---

done